azure/scope/clients.go

```diff
@@ -122,6 +122,16 @@ func (c *AzureClients) setCredentials(subscriptionID, environmentName string) er
 				Cloud: config,
 			},
 		}
+
+		// azidentity uses different envvars for certificate authentication
+		// Let's set it according to the envvars used by autorest, in case they are present
+		_, azidSet := os.LookupEnv("AZURE_CLIENT_CERTIFICATE_PATH")
+		path, autorestSet := os.LookupEnv("AZURE_CERTIFICATE_PATH")
+		if !azidSet && autorestSet {
+			os.Setenv("AZURE_CLIENT_CERTIFICATE_PATH", path)
+			os.Setenv("AZURE_CLIENT_CERTIFICATE_PASSWORD", os.Getenv("AZURE_CERTIFICATE_PASSWORD"))
+		}
+
```

Comment:
1. If we are going to go with this approach, please include a comment on what env var is supported by MSAL vs autorest. 

@CecileRobertMichon -- Thoughts?

Reply:
Done